src/routes/plugins-admin.js

```diff
@@ -75,6 +75,10 @@ function register(server, options) {
             return Boom.notFound();
         }
 
+        if (!api.githubToken) {
+            return Boom.notImplemented('configure_github_token');
```

Comment:
```suggestion
            return Boom.notImplemented('github-token-not-configured');
```
Think this would be more in line with existing error "keys" the API is using, like [https://github.com/datawrapper/api/blob/master/src/routes/auth.js#L559](here)

Reply:
Done. Thanks! I feel like the error messages in the API are all over the place and inconsistent. When there is some down time we should maybe find a common naming rule 😅. 

See [System for error codes (Notion)](https://www.notion.so/datawrapper/System-for-error-codes-23763fb19f1b494fb9437d295326fd51)